tests/manifests/user_scenarios/kitchen_sink.pp.erb

```diff
@@ -51,6 +51,7 @@ dsc_package { 'install_7zip':
   dsc_productid => '',
   dsc_arguments => "/S /D=${seven_zip_install_path}"
 }
+->
```

Comment:
This looks unrelated to the host... seems like a candidate for a separate maint PR